static/scss/partials/onboarding.scss

```diff
@@ -175,3 +174,235 @@
         }
     }
 }
+
+// Multi-part Premium Onboarding
+.is-premium-onboarding {
+
+    .c-multipart-premium-onboarding {
+        display: block;
+
+        @media #{$mq-md} {
+            display: flex;
+        }
+    }
+    
+    .dashboard-mpp-promo, .domain-alias-dashboard, .main-list, .mpp-dashbaord-header {
+        display: none;
+    }
+}
+
+.c-multipart-premium-onboarding {
+    display: none;
+    max-width: $content-lg;
+    margin-left: auto;
+    margin-right: auto;
+    position: relative;
+
+    @media #{$mq-md} {
+        min-height: calc(100vh - 150px);
```

Comment:
Maybe add a comment where the `150px` came from?

Reply:
🪄 Magic Number™

Reply:
```suggestion
        // 150px is an approx. size of the header and footer combined
        min-height: calc(100vh - 150px); 
```